src/services/utilServices/AuthService.js

```diff
@@ -9,7 +9,9 @@ const { ForbiddenError } = require("@errors/ForbiddenError")
 const validateStatus = require("@utils/axios-utils")
```

Comment:
i just realised this is also `AuthService` :monkas:

Reply:
Do you think we should also rename this service? We could file an issue (and there's also an `AuthService` under identity)

Reply:
tbh, our naming is p confusing so i feel like we should; i think filing an issue might be kicking the can down the road so if it's convenient here, i'm more inclined to doing it here (TIA HAHAHAH)

Reply:
It would probably involve frontend changes as well (endpoints are currently located in `/auth/...`) - hence why i think this would be better to handle in a separate PR

Reply:
sure, this sounds reasonable - could we file a tracking issue?

Reply:
Yup, issue filed https://github.com/isomerpages/isomercms-backend/issues/498